slack_rules/slack_passthrough_anomaly.yml

```diff
@@ -0,0 +1,80 @@
+AnalysisType: rule
+Filename: slack_passthrough_anomaly.py
+RuleID: Slack.AuditLogs.PassthroughAnomaly
+DisplayName: Slack Anomaly Detected
+Enabled: true
+LogTypes:
+  - Slack.AuditLogs
+Tags:
+  - Slack
+Severity: Critical
```

Comment:
Should this be critical? That seems a bit high for an anomaly detection IMO but I don't fully understand what anomaly means in the context of slack

Reply:
These are things that Slack tags as an anomaly per their [docs](https://api.slack.com/admins/audit-logs#anomalous-events)

This can include ASN being a known bad, anomalous IP detected, unexpected session fingerprint, connection from a Tor node, and changes in the user agent. 

I think this should be a Critical but perhaps it'd be better off as a High